playground/TestPlatform.Playground/Program.cs

```diff
@@ -53,16 +52,21 @@ static void Main(string[] args)
         var sources = new[] {
             Path.Combine(playground, "MSTest1", "bin", "Debug", "net472", "MSTest1.dll"),
             Path.Combine(playground, "MSTest1", "bin", "Debug", "net5.0", "MSTest1.dll"),
-            @"C:\Users\jajares\source\repos\TestProject48\TestProject48\bin\Debug\net48\TestProject48.dll",
-            @"C:\Users\jajares\source\repos\TestProject48\TestProject1\bin\Debug\net48\win10-x64\TestProject1.dll"
         };
 
         // console mode
         var settingsFile = Path.GetTempFileName();
         try
         {
             File.WriteAllText(settingsFile, sourceSettings);
-            var process = Process.Start(console, string.Join(" ", sources) + " --settings:" + settingsFile + " --listtests");
+            var processStartInfo = new ProcessStartInfo
+            {
+                FileName = console,
+                Arguments = string.Join(" ", sources, $"--settings:{settingsFile}", "--listtests"),
```

Comment:
There is a problem with the refactoring you did, `sources` will no longer be joined but considered as one item to join.

```suggestion
                Arguments = $"{string.Join(" ", sources)} --settings:{settingsFile} --listtests",
```